lgc/patch/NggPrimShader.cpp

```diff
@@ -953,15 +959,12 @@ void NggPrimShader::constructPrimShaderWithoutGs(Module *module) {
         //   ES_GS_OFFSET01[31:16] = vertexId1 (in dwords)
         //   ES_GS_OFFSET01[15:0]  = vertexId0 (in dwords)
 
-        // Use vertex0 as provoking vertex to distribute primitive ID
-        auto esGsOffset0 = m_builder->CreateIntrinsic(Intrinsic::amdgcn_ubfe, m_builder->getInt32Ty(),
-                                                      {
-                                                          m_nggFactor.esGsOffsets01,
-                                                          m_builder->getInt32(0),
-                                                          m_builder->getInt32(16),
-                                                      });
+        const auto esGsRingItemSize =
```

Comment:
I think 'unsigned' is much clearer than 'auto' here, so we can easily know it is a constant value.

Reply:
OK. I will correct this.